src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -54,7 +54,10 @@ public static async Task<ContributionProvider> Create(Docset docset, GitHubUserC
             var userIds = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
             var updatedDateTime = GetUpdatedAt(document, commits);
 
-            var resolveGitHubUsers = GitHubUtility.TryParse(repo?.Remote, out var gitHubOwner, out var gitHubRepoName) && document.Docset.Config.GitHub.ResolveUsers;
+            _ = GitHubUtility.TryParse(repo?.Remote, out var gitHubOwner, out var gitHubRepoName) ||
+                GitHubUtility.TryParse(document.Docset.Config.Contribution.Repository, out gitHubOwner, out gitHubRepoName);
+
+            var resolveGitHubUsers = document.Docset.Config.GitHub.ResolveUsers;
```

Comment:
we only resolve github users if both (resolve users is true) and (the repository or contribution repository) is github like, this change omits the second condition and will cause unneeded github api calls

Reply:
make senses